dask/async.py

```diff
@@ -429,6 +417,10 @@ def get_async(apply_async, num_workers, dsk, result, cache=None,
     """
     assert queue
 
+    if callbacks is None:
+        callbacks = _globals['callbacks']
+    callbacks = setup_callbacks(callbacks)
```

Comment:
It certainly could, but a check for `None` is certainly faster than calling a no-op function. By doing it the current way, if there are 3 `callbacks` objects, none of which have `posttask` callback, then in the scheduler loop it hits the `if callbacks.posttask` statement (which is False), and continues on. An equivalent option would be to normalize to 4 lists, and do the iteration in the scheduler. This fixes my concerns, while also avoiding the (small) cost of creating functions.